core/string/string_name.h

```diff
@@ -132,10 +132,15 @@ class StringName {
 		return _data >= p_name._data;
 	}
 	_FORCE_INLINE_ bool operator==(const StringName &p_name) const {
-		// the real magic of all this mess happens here.
-		// this is why path comparisons are very fast
+		// The real magic of all this mess happens here.
+		// This is why path comparisons are very fast.
 		return _data == p_name._data;
 	}
+	_FORCE_INLINE_ bool operator!=(const StringName &p_name) const {
+		// The real magic of all this mess happens here.
+		// This is why path comparisons are very fast.
```

Comment:
I think the comment is unnecessary, if there is exact same one just above.

Reply:
I was thinking the same; I've removed the lower comment but kept the change to the above one.